cc/gtp_player.cc

```diff
@@ -290,34 +292,17 @@ GtpPlayer::Response GtpPlayer::HandleGamestate(absl::string_view cmd,
     }
     oss << ch;
   }
-  std::string board = oss.str();
-
-  // toPlay field.
-  std::string to_play = position.to_play() == Color::kBlack ? "Black" : "White";
-
-  // lastMove field.
-  std::string last_move;
+  j["toPlay"] = position.to_play() == Color::kBlack ? "B" : "W";
```

Comment:
Can you use the format below

nlohmann::json j = {
      {"moveNum", pos.n()},
      {"toPlay", pos.to_play() == Color::kBlack ? "B" : "W"},
    ...
  };

for all the properties (minus lastMove)?